superset/assets/javascripts/explorev2/main.css

```diff
@@ -25,3 +25,8 @@
     margin-top: 0px;
     margin-right: 3px;
 }
+
+.control-panel-section {
```

Comment:
if there was already a parent class, we could reference it the way you are describing, but since there is not i decided to make it more specific and make a new class for these control panel sections. there's not a big difference in approaches.